src/command/layout.zig

```diff
@@ -0,0 +1,35 @@
+// This file is part of river, a dynamic tiling wayland compositor.
+//
+// Copyright 2020 Leon Henrik Plickat
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+const c = @import("../c.zig");
+
+const Arg = @import("../Command.zig").Arg;
+const Seat = @import("../Seat.zig");
+
+pub fn defaultLayout(seat: *Seat, arg: Arg) void {
+    const layout_name = arg.str;
+    var config = seat.input_manager.server.config;
+    config.layout = seat.focused_output.getLayoutByName(layout_name);
+}
```

Comment:
Hmm, I just realized that this is never used. Maybe we should just get rid of it. (also this modifies a copy of `server.config` not the member itself, but that's irrelevant if we remove it)

Reply:
The idea behind that command is to allow changing the default layout. If the user can not change it, all new outputs will start with the LeftMaster layout.

Reply:
Although thinking about it, that command is probably flawed anyway: If it gets executed in a `river-startup.sh` script or something similar, all outputs are already connected...

Reply:
That makes sense, though in it's current state it doesn't seem to do that.

Reply:
I'll remove it. Changing the default layout of outputs can probably be done with a script that listens for river events and automatically changes the layout.

Reply:
Done. I also removed the `layout` variable from the `Config` class. Since the default layout is no longer configurable, it does not need to be in `Config`.